src/inference/src/cache_manager.hpp

```diff
@@ -149,12 +148,8 @@ class FileStorageCacheManager final : public ICacheManager {
     void remove_cache_entry(const std::string& id) override {
         auto blobFileName = getBlobFile(id);
 
-        if (ov::util::file_exists(blobFileName)) {
-#if defined(_WIN32) && defined(OPENVINO_ENABLE_UNICODE_PATH_SUPPORT)
-            _wremove(blobFileName.c_str());
-#else
-            std::remove(blobFileName.c_str());
-#endif
+        if (std::filesystem::exists(blobFileName)) {
+            (void)std::filesystem::remove(blobFileName);
```

Comment:
Is it required here also for new function?
```suggestion
            std::ignore = std::filesystem::remove(blobFileName);
```

Reply:
suggestion applied